tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -25,6 +25,7 @@ public class RecordingHandler
         #region constructor and common variables
         public string CurrentBranch = "master";
         public string RepoPath;
+        private const string SkipRecordingHeader = "x-recording-skip";
```

Comment:
To keep these from evolving into various permutations in the future, would it make sense to make these kind of config choices distinct from `EntryRecordMode` and rename it to something like `x-recording-config` allowing multiple values?

I could imagine us wanting to send a request like this someday:
`x-recording-config=ignore-x, ignore-y, delete-body`

Which would be used in `EntryRecordMode.Record`

Reply:
I considered doing something more generic, but there are already ways to do things like ignore headers, sanitize body, etc. This has a very specific purpose so I wanted to keep it scoped as such.

Reply:
@scbedd what do you think?

Reply:
Right now, we have two types of ignore.

1. Ignore recording the request entirely
2. Ignore recording the body only

We currently provide OTHER "ignore" functionality by using sanitization. I'm down with Chris's feedback, but I only think it's worth the additional effort if we have another thing to ignore. Given that I don't see that extending, I don't necessarily think it's worth to do a plaintext string -> Flags mapping.

Reply:
Yeah I don't really see other things that we will need to ignore recording in the future.